# start must have both position and orientation.
# the orientation of end is not used.
Header header
geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped end